app/assets/javascripts/discourse/app/components/post-list/item/details.gjs

```diff
@@ -1,65 +1,95 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
 import { hash } from "@ember/helper";
 import { htmlSafe } from "@ember/template";
 import PluginOutlet from "discourse/components/plugin-outlet";
+import TopicStatus from "discourse/components/topic-status";
 import categoryLink from "discourse/helpers/category-link";
 import getURL from "discourse/lib/get-url";
 import { prioritizeNameInUx } from "discourse/lib/settings";
 import { i18n } from "discourse-i18n";
 
 export default class PostListItemDetails extends Component {
-  get titleAriaLabel() {
+  @tracked
+  url =
+    this.args.post?.url || this.args.post?.post_url || this.args.post?.postUrl;
```

Comment:
Technically, the `@post` argument can change after the component is first initialized, so this should be a getter rather than a field initializer

(also, storing less state is always better for reducing bug surface-area!)